### PR TITLE
[8.0]-IMP-Stages as clickable in BR

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
#223 
- Stages as clickable in BR main module
- Removed code unnecessary code.

### CODE REVIEW

**Comment by elicoidal** [comment]
Please check travis

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal Can you please review.

**Review by elicoidal — APPROVED** [review]
Functional tests OK

**Comment by elicoidal** [comment]
@victormartinelicocorp @seb-elico

File: business_requirement/models/business.py [hunk 1]
```diff
@@ -353,6 +316,74 @@ def message_post(self, cr, uid, thread_id, body='', subject=None,
         )
         return res
 
+    @api.model
+    def read_group(self, domain, fields, groupby, offset=0,
```
Comment (martintorresvm) [inline business_requirement/models/business.py:320]: @YogeshMahera-SerpentCS Add docstring:

- [x] Short functionality description
- [ ] Input
- [ ] Ouput

**Review by martintorresvm — CHANGES_REQUESTED** [review]

**Review by seb-elico — APPROVED** [review]
LGTM after adding docstring

**Comment by YogeshMahera-SerpentCS** [comment]
@victormartinelicocorp 
Added docstring

        """ Read group customization in order to display all the stages in the
            kanban view. if the stages values are there it will group by state.
        """

**Review by martintorresvm — APPROVED** [review]

**Pull request merged**